dbt/adapters/dremio/connections.py

```diff
@@ -231,6 +230,79 @@ def create_catalog(self, relation):
             logger.debug(f"Creating folder(s): {database}.{schema}")
             self._create_folders(database, schema, rest_client)
         return
+    
+    # dbt docs integration with Dremio wikis and tags
+    def docs_integration_with_wikis(self, relation, text: str):
```

Comment:
All functions should be name as verbs. 
Let's name this function as a  process_wikis()